Formula/react-native-cli.rb

```diff
@@ -0,0 +1,20 @@
+require "language/node"
+
+class ReactNativeCli < Formula
+  desc "The React Native CLI tools"
+  homepage "https://github.com/facebook/react-native#readme"
+  url "https://registry.npmjs.org/react-native-cli/-/react-native-cli-2.0.1.tgz"
+  sha256 "f1039232c86c29fa0b0c85ad2bfe0ff455c3d3cd9af9d9ddb8e9c560231a8322"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    output = shell_output("#{bin}/react-native -h")
```

Comment:
We need a test that exercises the some of the functionality of the app.

Reply:
The only thing this app can do until you initialize a new project is initialize one, which is a rather lengthy process. But I've updated the test to do just that.